presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -242,7 +242,10 @@ void PrestoServer::run() {
 
   velox::dwrf::registerDwrfReaderFactory();
 #ifdef PRESTO_ENABLE_PARQUET
-  velox::parquet::registerParquetReaderFactory();
+  velox::parquet::registerParquetReaderFactory(
+      systemConfig->parquetReaderType() == "native" ?
```

Comment:
Instead of adding more configuration should we just change the default to native Parquet reader and not use DuckDB reader at all? We are going to remove DuckDB reader from Velox soon.